config/default/common/config/wv.json/layers/viirs/VIIRS_SNPP_DayNightBand_ENCC.json

```diff
@@ -3,7 +3,7 @@
     "VIIRS_SNPP_DayNightBand_ENCC": {
       "id": "VIIRS_SNPP_DayNightBand_ENCC",
       "title": "Nighttime Imagery (Day/Night Band, Enhanced Near Constant Contrast)",
-      "subtitle": "Suomi NPP / VIIRS",
+      "subtitle": "Suomi NPP/VIIRS",
```

Comment:
![Screen Shot 2021-05-04 at 1 42 33 PM](https://user-images.githubusercontent.com/1480000/117046357-9a842200-acde-11eb-8d18-0ca287292430.png)

I think we want the spaces?

These are the other affected layers:
![Screen Shot 2021-05-04 at 1 43 26 PM](https://user-images.githubusercontent.com/1480000/117046479-bbe50e00-acde-11eb-9494-14feb05b2162.png)

Reply:
oh yeah, I couldn't remember if we were doing spaces or not.

Reply:
spaces added